packages/commuter-cli/src/index.js

```diff
@@ -23,8 +25,8 @@ const symlink = (target, path, type = "dir") => {
 
 const startCommuterServer = () => {
   symlink(
-    `${INSTALL}/commuter-server/src/build`,
-    `${INSTALL}/commuter-client/build`
+    `${INSTALL_DIR}/${COMMUTER_SERVER}/src/build`,
+    `${INSTALL_DIR}/${COMMUTER_CLIENT}/build`
```

Comment:
Will these then be `${INSTALL_DIR/@nteract/commuter-server/src/build`? Is that what was intended?